dev-requirements.txt

```diff
@@ -1,2 +1,8 @@
-pytest == 3.*
-nose >= 1.3
+# Pin exact dependency version numbers for local dev and CircleCI.
+numpy==1.19.0 ; python_version > '3.5'
```

Comment:
How about we drop 3.5 support now since sklearn, numpy, etc. already have and since 3.5 is about to stop getting security fixes [in september](https://en.wikipedia.org/wiki/History_of_Python#Table_of_versions)?

Reply:
I wouldn't mind helping to move the community forward! Just dropped python 3.5 at c3d9919